code/modules/food_and_drink/snacks.dm

```diff
@@ -249,6 +250,48 @@
 		..()
 		src.add_topping(0)
 
+/obj/item/reagent_containers/food/snacks/pizza/fresh
+    name = "fresh pizza"
+    desc = "A cheesey pizza pie with thick tomato sauce."
+    icon_state = "cheesy"
+    sliced_icon = "cheesy-slice"
+	
+/obj/item/reagent_containers/food/snacks/pizza/ball
```

Comment:
Correct me if I'm wrong, but are /pizza/fresh, /pizza/ball, /pizza/shroom,  and /pizza/peper all unobtainable, since they don't appear in the /obj/random_item_spawner/pizza?

Reply:
Yes pizza fresh is the same sprite as the new pizza and the rest are sprites I didnt want to go to waste so I left them in there thinking perhaps someone more creative  then I could find a good use for them.

Reply:
i would like to see them used somewhere

Reply:
done'd